src/obb/server/interactors/create_lobby_game.clj

```diff
@@ -0,0 +1,19 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.interactors.create-lobby-game
```

Comment:
I associated this create-game with the lobby because we will have several create game scenarios: lobby, friendly challenge, tournament. We may be revealing implementation details, but more importantly I believe we are showing intent and semantics. Do you agree?

About the interactors, is your issue with using the *interactor* concept? How could we represent this with something else? (a fn?) Because an interactor fn will have several side utillity fns and it's nice to group them on a "small" namespace.

Why do you think the create game is the main namespace? Why not the lobby or the play? Currently the of this server module is responsible for booting the server.